package.json

```diff
@@ -30,6 +30,7 @@
     "@ember/edition-utils": "^1.2.0",
     "@ember/optional-features": "^2.0.0",
     "@ember/render-modifiers": "^1.0.2",
+    "@ember/string": "^1.0.0",
```

Comment:
https://github.com/emberjs/ember-string/tree/v2.0.0 is tagged, is it not released to npm?

Reply:
Seems like I didn't publish. I don't remember if I had any specific reason not to 🤔 Maybe I was waiting for the toc deprecation?